apps/roboshield/src/payload/Blocks/RoboForm.ts

```diff
@@ -0,0 +1,412 @@
+      type: "group",
+      fields: [
+        {
+          name: "label",
+          type: "text",
+          required: true,
+          defaultValue: "Search Engine Crawlers",
+        },
+        {
+          name: "title",
+          type: "text",
+          required: true,
+          defaultValue:
+            "Search engine crawlers are used to index web content for search engines. Blocking them could prevent your website from being discovered by users.",
+        },
+      ],
+    },
+  ],
+};
+
+const SiteMaps: Block = {
+  slug: "site-maps",
+  labels: { singular: "Site Maps", plural: "Site Maps" },
+  fields: [
+    {
+      name: "title",
+      type: "text",
+      required: true,
+      defaultValue: "Site Maps",
+    },
+    richText({
+      name: "hint",
+      defaultValue: [
+        {
+          children: [
+            { text: "You can add sitemap URLs to your robots.txt file." },
+          ],
+        },
+      ],
+    }) as unknown as Field,
+    {
+      name: "placeholder",
+      type: "text",
+      required: true,
+      defaultValue: "Enter sitemap URLs each URL on a new line",
+    },
+  ],
+};
+
+const Finish: Block = {
+  slug: "finish",
+  labels: { singular: "Finish", plural: "Finish" },
+  fields: [
+    {
+      name: "title",
+      type: "text",
+      required: true,
+      defaultValue: "Finish",
+    },
+    richText({
+      name: "hint",
+      defaultValue: [
+        {
+          children: [
+            {
+              text: "Your robots.txt file has been generated successfully. You can now copy the code or download the file.",
+            },
+          ],
+        },
+      ],
+    }) as unknown as Field,
+    {
+      name: "placeholder",
+      type: "text",
+      required: true,
+      defaultValue:
+        "# Disallow specifies the paths that are not allowed to be crawled by the robot.",
+    },
+  ],
+};
+
+const Labels: Field = {
+  name: "labels",
+  label: "Labels",
+  type: "group",
+  fields: [
+    {
+      name: "continue",
+      type: "text",
+      required: true,
+      defaultValue: "Continue",
+    },
+    {
+      name: "back",
+      type: "text",
+      required: true,
+      defaultValue: "Back",
+    },
+    {
+      name: "reset",
+      type: "text",
+      required: true,
+      defaultValue: "Reset",
+    },
+    {
+      name: "download",
+      type: "text",
+      required: true,
+      defaultValue: "Download",
+    },
+    {
+      name: "copyToClipboard",
+      type: "text",
+      required: true,
+      defaultValue: "Copy to Clipboard",
+    },
+  ],
+};
+const RoboForm: Block = {
+  slug: "robo-form",
+  labels: { singular: "Robo Form", plural: "Robo Form" },
+  fields: [
+    {
+      name: "toolTipText",
+      type: "text",
+      required: true,
+      defaultValue: "View current robots.txt file",
+    },
+    {
+      type: "blocks",
+      name: "steps",
+      blocks: [ExistingRobots, Delays, Paths, BlockBots, SiteMaps, Finish],
```

Comment:
@koechkevin I believe these should be there by default, rather than allowing users to select which ones to display

Reply:
@kelvinkipruto https://github.com/CodeForAfrica/ui/pull/746/files#r1662115577